youtube-homepage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles.css">
+  <link rel="shortcut icon" href="/assets/shortcut.png">
+  <title>YouTube</title>
+</head>
+
+<body>
+  <nav class="nav-bar">
+  <div id="nav1">
+    <img id="img-logo" src="/assets/yt-logo.png" alt="youtube-logo">
+    <input id="search-bar" type="text" placeholder="Search...">
+    <button id="search-button" type="submit">Search</button>
+  </div>
+  <div id="nav2">
+    <img id="create-img" src="/assets/create.png" alt="create">
+    <img id="apps-img" src="/assets/apps.png" alt="apps">
+    <img id="notifications-img" src="/assets/notifications.png" alt="notifications">
+  </div>
+  </nav>
+  <div class="split-left" >
+  
+  <section class="video-section">
+    <iframe class="responsive-iframe" src="https://www.youtube.com/embed/HbSjyU2vf6Y" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <h1>Git & GitHub Tutorial for Beginners by Net Ninja</h1>
+        <ul>
+          <li><a href="#">Likes</a>
+            <img src="assets/1.png" alt="likes"></li>
+          <li><a href="#">Dislikes</a>
+            <img src="/assets/2.png" alt="dislikes"></li>
+          <li><a href="#">Share</a>
+             <img src="/assets/3.png" alt="share"></li>
+          <li class="views"><p>66.341 Views</p></li>
+        </ul>
+  </section>
+
+  <article class="description">
+    <img class="avatar-img" src="/assets/todp-img.png" alt="avatar">
+    <input id="subscribe-btn" type="image" src="/assets/subscribe.png" alt="subscribe">
+    <br>
+    <p>Hey ninjas, in this Git & GitHub tutorial I want to show you 
+      how you can fork a repository to your own GitHub account. You'd 
+      typically do this if you wanted to contribute to some kind of 
+      open source project.<p>
+      <br>
+      <p>---------------------------------------------------------------------------------------------</p>
```

Comment:
This is not a good practice. You can use an horizontal line `<hr>` for this. You just style it up with your CSS to look like this. Besides, I'm not sure this is in the project.